### ISSUE TITLE
UML exercise on Person, Inbox, Message: Why no arrows?

### ISSUE THREAD

**Issue by damithc** [nus-cs2103-AY2122S1/forum#296]
Question received from a student: In the following video (found in https://nus-cs2103-ay2122s1.github.io/website/schedule/week8/topics.html), why the associations are shown as lines, not arrows?
Shouldn't the Person reference Inbox and Inbox could also reference the owner of the inbox?

![image](https://user-images.githubusercontent.com/1673303/136495756-47facb42-f929-419a-89c9-c31b3a247299.png)

Exercise question, for reference (ignore the highlighting): 
![image](https://user-images.githubusercontent.com/1673303/136495711-fc656869-6e1e-42f6-91a1-0537203097cd.png)

**Comment by AllardQuek** [comment]
Since the question states "A person's Inbox" should it be an arrow from Person to Inbox? Meaning a Person has a reference to an Inbox but an Inbox will not have a reference to a Person.

**Comment by Javiier-pzk** [comment]
I think no arrows are shown because the question asked us to ignore `navigabilities`

**Comment by damithc** [comment]
> I think no arrows are shown because the question asked us to ignore `navigabilities`

Good point @Javiier-pzk For arguments' sake, let's say the question _didn't_ ask us to ignore navigability. What then? Thoughts from others welcome.

**Comment by darrenhoon** [comment]
Think it would have been just an association line without arrows as the question did not hint as to how the Person-Inbox relationship is implemented. For example, it could be that the Inbox strictly holds only 1 Person reference in it's implementation while Person has no reference to Inbox at all, which would suggest navigability from Inbox to Person as opposed to Person to Inbox (as counter-intuitive as it may sound). This would still satisfy the requirement of "A Person's Inbox". But since such a scenario was not presented, the information regarding navigability is not present so my take would be that it would still be an association line without arrows.

Could be wrong but please feel free to add on!

**Comment by damithc** [comment]
> Think it would have been just an association line without arrows as the question did not hint as to how the Person-Inbox relationship is implemented. For example, it could be that the Inbox strictly holds only 1 Person reference in it's implementation while Person has no reference to Inbox at all, which would suggest navigability from Inbox to Person as opposed to Person to Inbox (as counter-intuitive as it may sound). This would still satisfy the requirement of "A Person's Inbox". But since such a scenario was not presented, the information regarding navigability is not present so my take would be that it would still be an association line without arrows.

Good answer @darrenhoon 

This is also why we omit navigability in OODM diagrams to be covered in week 9 i.e., navigability is an implementation decision which may not be known yet.
